app/views/users/show.html.erb

```diff
@@ -1,2 +1,23 @@
-<h1>Here you will find a specific user. The page will display user name, number of posts, bio and a summary of the posts</h1>
+<div class="user-details">
+  <div class="user-container">
+    <div class="user-img black-border">
+      <%= image_tag("icon.png") %>
+    </div>
+    <div class="user-overview black-border">
+      <h2><%= @user.name %></h2>
+      <span>Number of posts: <%= @user.posts_counter || 0 %></span>
+    </div>
+  </div>
+  <div class="user-bio black-border">
+    <span>Bio</span>
+    <p><%= @user.bio %></p>
+  </div>
+  <% @user.posts.each do |post| %>
+    <div class="post-overview black-border">
+      <h3><%= link_to post.title, user_posts_path(post) %></h3>
+      <p><%= post.text %></p>
+      <span>Comments: <%= post.comments_counter || 0 %>, Likes: <%= post.likes_counter || 0 %></span>
+    </div>
```

Comment:
- [ ] According to the diagram, there should be a button `See all post` at the bottom of the user show page which leads to the post index where you see all the user's `posts`. This button might look something like this 
```ruby
<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```
<img width="311" alt="chrome_ecBQjnmFss" src="https://user-images.githubusercontent.com/81468675/155400307-94afa81e-5a1c-4931-9d08-5a7672941125.png">

Reply:
Added!